src/main/java/com/commercetools/sync/services/impl/TypeServiceImpl.java

```diff
@@ -1,22 +1,24 @@
 package com.commercetools.sync.services.impl;
 
 
+import com.commercetools.sync.commons.utils.CtpQueryUtils;
 import com.commercetools.sync.services.TypeService;
 import io.sphere.sdk.client.SphereClient;
-import io.sphere.sdk.queries.QueryExecutionUtils;
+import io.sphere.sdk.types.Type;
 import io.sphere.sdk.types.queries.TypeQuery;
 
 import javax.annotation.Nonnull;
+import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Consumer;
 
 /**
  * Implementation of TypeService interface.
- * TODO: USE graphQL to get only keys OR MAKE PR/ISSUE TO FIX QueryExecutionUtils.queryAll
- * TODO: INTEGRATION TEST GITHUB ISSUE#7
+ * TODO: USE graphQL to get only keys.
```

Comment:
@lojzatran nope, it would be just more efficient due to having smaller payloads.